modules/local/append_clusters/main.nf

```diff
@@ -0,0 +1,21 @@
+process APPEND_CLUSTERS {
+    tag "Append additional clusters from database"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/csvtk:0.22.0--h9ee0642_1' :
+        'biocontainers/csvtk:0.22.0--h9ee0642_1' }"
+
+    input:
+    path(initial_clusters)
+    path(additional_clusters)
+
+    output:
+    path("reference_clusters.tsv")
+
+    script:
+    """
+    csvtk concat -t ${initial_clusters} ${additional_clusters} > reference_clusters.tsv
```

Comment:
Do you think it would be possible to do some additional checking here to make sure all the clusters have addresses with the same number of levels? I don't know if there is some other csvtk command that could be used to check for this?

If it would be difficult, we could leave this out right now, but I would ideally like additional error checking done to make sure files are all consistent.

Reply:
I think I figured out a way to do this, but I am going to save it for the next PR as I will be making changes to the formatting of the `--db_clusters` and `cluster_file` modules to align with the changes made to `GAS CALL`.

Reply:
Thanks so much for this Kyla 😄